lib/controller.js

```diff
@@ -24,22 +24,40 @@ var responses = {
 
 // Wrapper function for Tessel.list to set SSH key path
 controller.list = function(opts) {
-  return controller.keyHelper(opts, Tessel.list);
+  return controller.defaultHelpers(opts, Tessel.list);
 };
 
 // Wrapper function for Tessel.get to set SSH key path
 controller.get = function(opts) {
-  return controller.keyHelper(opts, Tessel.get);
+  return controller.defaultHelpers(opts, Tessel.get);
 };
 
-controller.keyHelper = function(opts, next) {
+// Calls any helpers for the get and list codepaths
+controller.defaultHelpers = function(opts, next) {
+  // First disable output if necessary
+  return controller.outputHelper(opts)
+    // Then make sure our SSH Key path is updated
+    .then(() => controller.keyHelper(opts))
+    // before continuing to the next function
+    .then(() => next(opts));
+};
+
+controller.keyHelper = function(opts) {
```

Comment:
Since there are two functions every command should call (`keyHelper` and `outputHelper`), I decided it made sense to just refactor into a single function that would be easier to build upon.